doc/nrf/protocols/lte/psm.rst

```diff
@@ -0,0 +1,267 @@
+.. _cellular_psm:
+
+Power saving techniques
+#######################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Cellular technology is an example of an LPWAN (Low Power Wide Area Network) because to its coverage, high mobility, roaming capabilities, and infrastructure.
+However, battery lifespan can be a problem for devices using cellular IoT if you do not fully understand the power-saving capabilities and how to use them to decrease power consumption.
+
+This guide explains following concepts for saving power:
+
+* :ref:`Radio activity modes <cellular_radio_modes>`, to enable you to examine power-saving modes in cellular IoT.
```

Comment:
I would rather talk about LTE here as well. Even if the top heading was changed from LTE to Cellular, when talking about LTE features, we need to still refer to LTE. Cellular covers everything starting from analog networks (1G) to 6G, so basically the only thing in common is the use of radio communication on a cell based system. Maybe it would have been better starting point to rename only the top level heading.

Reply:
LTE-M and NB-IoT are ok. I probably I misunderstood what "to enable you to examine power-saving modes" here meant, I guess it just means that the radio activity modes are presented here to provide context.